checkout-v3/index.md

```diff
@@ -78,7 +82,13 @@ this section. We recommend [getting started here][get-started].
          </span>
          <span class="cards-content">
             <span class="h4">One-Click Payments</span>
-            <span><p>Save your customer's details and make their checkout faster.</p>
+            <span>
+               <p>Save your customer's details and make their checkout faster.</p>
+               <span class="badge badge-gray-light">Card</span>
```

Comment:
The code `<span class="badge badge-gray-light">Payment method name</span>` result in a grey badge on the card, shown in the screenshot below

![image](https://github.com/user-attachments/assets/dbe2a54a-bf55-43d7-85ab-3d958a550dc4)